ecf/scripts/gdas/atmos/analysis/jgdas_atmos_analysis_wdqms.ecf

```diff
@@ -0,0 +1,63 @@
+#PBS -S /bin/bash
+#PBS -N %RUN%_atmos_analysis_wdqms_%CYC%
+#PBS -j oe
+#PBS -q %QUEUE%
+#PBS -A %PROJ%-%PROJENVIR%
+#PBS -l walltime=00:10:00
+#PBS -l select=1:mpiprocs=1:ompthreads=1:ncpus=1:mem=48GB
+#PBS -l place=vscatter:shared
+#PBS -l debug=true
+
+export model=gfs
+%include <head.h>
+%include <envir-p1.h>
+
+set -x
+
+export NET=%NET:gfs%
+export RUN=%RUN%
+export CDUMP=%RUN%
+
+############################################################
+# Load modules
+############################################################
+module load PrgEnv-intel/${PrgEnv_intel_ver}
+module load craype/${craype_ver}
+module load intel/${intel_ver}
+module load cray-mpich/${cray_mpich_ver}
+module load cray-pals/${cray_pals_ver}
+module load hdf5/${hdf5_ver}
+module load netcdf/${netcdf_ver}
+# TODO: Needs the python modules. Review the above list (hdf5 and netcdf)
+module list
+
+#############################################################
+# WCOSS environment settings
+#############################################################
+export cyc=%CYC%
+export cycle=t%CYC%z
+
+############################################################
+# CALL executable job script here
+############################################################
+${HOMEgfs}/jobs/JGDAS_ATMOS_ANALYSIS_WDQDMS
+if [ $? -ne 0 ]; then
+   ecflow_client --msg="***JOB ${ECF_NAME} ERROR RUNNING J-SCRIPT ***"
+   ecflow_client --abort
+   exit
+fi
+
+%include <tail.h>
+%manual
+######################################################################
+# Purpose: To execute the job that creates the WDQMS products
+######################################################################
+
+######################################################################
+# Job specific troubleshooting instructions:
+#  see generic troubleshoot manual page
+#
+######################################################################
+
+# include manual page below
```

Comment:
This is new. Is this something we should be adding to all ecf scripts?

Reply:
yes. providing tooltip information in the ecflow UI on what the job is, is a good thing.